container/lxd/initialisation_linux.go

```diff
@@ -22,10 +22,10 @@ import (
 	"github.com/juju/utils/packaging/manager"
 	"github.com/juju/utils/proxy"
 	"github.com/juju/utils/set"
-	"github.com/lxc/lxd/shared"
 
 	"github.com/juju/juju/container"
-	"github.com/juju/juju/tools/lxdclient"
+	"github.com/juju/juju/network"
+	"github.com/lxc/lxd/shared/api"
```

Comment:
this import should be in the above set, because it is not a "github.com/juju/juju" namespace